MediatorHandler/LoginHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MediatR;
+using MVCCore;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class LoginHandler : IRequestHandler<Login,bool>
+{
+	public LoginHandler()
+	{
+
+	}
+    public bool ValidateUser(string UserName, string Password)
+    {
+        if (UserName == "Harpreet" && Password == "1234")
```

Comment:
Handler will only use to route the request, No business logic should be written inside them